druid/examples/async_event.rs

```diff
@@ -75,62 +70,22 @@ fn generate_colors(event_sink: druid::ExtEventSink) {
             (_, _) => Color::rgb8(r, g, b.wrapping_add(3)),
         };
 
-        // We send a command to the event_sink. This command will be
-        // send to the widgets, and widgets or controllers can look for this
-        // event. We give it the data associated with the event and a target.
-        // In this case this is just `Target::Auto`, look at the identity example
-        // for more detail on how to send commands to specific widgets.
-        if event_sink
-            .submit_command(SET_COLOR, color.clone(), Target::Auto)
-            .is_err()
-        {
-            break;
-        }
+        let color_clone = color.clone();
+        // schedule idle callback to change the data
+        event_sink.add_idle_callback(move |data: &mut Color| {
+            *data = color_clone;
```

Comment:
I think this is totally the right way to do this, but maybe the previous code had some pedagogical value? Sometimes you really do want to send a command instead of mutate the data...

Reply:
I feel blocking_function example is sufficient for that :man_shrugging: